dockerfiles/alpine/Dockerfile

```diff
@@ -27,6 +28,14 @@ COPY bob-apps/Cargo.toml bob-apps/Cargo.toml
 COPY bob-access/Cargo.toml bob-access/Cargo.toml
 RUN sed -i "s|\[\[bench\]\]|\[\[bench_ignore\]\]|g" */Cargo.toml
 
+# estimate build directory
+RUN echo "$(case "$BUILD_PROFILE" in\
+  ("dev") echo "debug";;\
+  ("test") echo "debug";;\
+  ("bench") echo "release";;\
+  (*) echo "$BUILD_PROFILE";;\
+  esac)" >> ./build_profile_dir
```

Comment:
Why do you not use environment variable here?

Reply:
Also I noticed that `ENV OUT_DIR` is declared above but not used. Probably, you should declare `ENV BUILD_PROFILE_DIR` instead and set its value in that command